docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
In `/etc/os-release` I see `Debian GNU/Linux stretch/sid`. I think that this will change to `<v10 codename>/sid` after stretch is released. Maybe `stretch/sid` is too confusing though and it will be read as in both stretch and sid. Maybe we shouldn't include this addition at all, or perhaps we can just say `Debian stretch` as I believe that is what @conorsch is running and provides a more stable point of reference.